### ISSUE TITLE
Do not auto select first option and populate field

### ISSUE THREAD

**Issue by martincarstens** [JedWatson/react-select#2653]
Current behavior: 

-- start typing charachers
-- dropdown shows partial matches and first result is automatically highlighted
-- hit enter, and first result populates search box

Desired behavior (via attribute):
-- start typing charachers
-- dropdown shows partial matches and first result is not automatically highlighted
-- press down arrow
-- first result is highlighted
-- hit enter, and first result populates search box

Basically the behavior prior to the bellow issue
https://github.com/JedWatson/react-select/issues/2

Is this possible currently?

**Comment by trkaplan** [comment]
"Auto selecting first option" must be optional.
Because I use Select component mainly as a Search Box. 
I want to get the typed input, after hitting enter key.  It'll be still possible to select one an option from the list.

**Comment by noahzweben** [comment]
@martincarstens did you figure this out?

**Comment by martincarstens** [comment]
I did, but had to edit source.

It looks like you won't have to wait too long for this option however:
https://github.com/JedWatson/react-select/pull/3484

The source to be merged is here:
https://github.com/JedWatson/react-select/pull/3484/files/3cdaf4669d649714852a7dca560dd782cc19076f

m
